downstream/modules/platform/proc-edge-manager-install-CLI.adoc

```diff
@@ -0,0 +1,26 @@
+[id="edge-manager-install-CLI"]
+
+= Installing the {RedHatEdge} CLI
+
+To install the {RedHatEdge} CLI, complete the following steps:
+
+.Procedure 
+
+. Enable the subscription manager for the repository appropriate for your system by running the following command:
+//The following command is for ACM, confirm steps for AAP.
++
+[source,bash]
+----
+subscription-manager repos --enable rhacm-2.13-for-rhel-<version>-<arch>-rpms
+----
+
++
+For a full list of available repositories for the {RedHatEdge}, see the [ADD LINK]_Additional resources_ section.
+
+. Install the `flightctl` CLI with your package manager. Run the following command:
```

Comment:
Suggest changing to:

Install the `flightctl` CLI with your package manager by running the following command: